src/vm-vcpu/src/vm.rs

```diff
@@ -245,7 +249,9 @@ mod tests {
         // We're specifying in the VmState that we want to run one vcpu, but we do not create
         // any. This should return an error.
         let (mut vm, _guest_memory) = default_vm(1).unwrap();
-        assert!(matches!(vm.run(), Err(Error::RunVcpus(e)) if e.kind() == ErrorKind::InvalidInput));
+        assert!(
+            matches!(vm.run(GuestAddress(0x10_000_000)), Err(Error::RunVcpus(e)) if e.kind() == ErrorKind::InvalidInput)
```

Comment:
nit: I think clippy is suggesting to group the bytes like this: 0x1000_0000

Reply:
Clippy doesn't seem to care a lot about the grouping, just that the group is not too large:
- https://rust-lang.github.io/rust-clippy/master/#large_digit_groups
- https://rust-lang.github.io/rust-clippy/master/#inconsistent_digit_grouping